Internal/Debug App/Tests/Unit Tests/v2/HeadlessVaultManagerTests.swift

```diff
@@ -335,7 +335,8 @@ final class HeadlessVaultManagerTests: XCTestCase {
                                         XCTAssert(true, "Failed with error \(err.localizedDescription)")
                                         
                                     } else if let updatedVaultedPaymentMethods = updatedVaultedPaymentMethods {
-                                        XCTAssert(!updatedVaultedPaymentMethods.isEmpty, "Failed to delete vaulted payment method")
+                                        XCTAssert(updatedVaultedPaymentMethods.isEmpty, "Failed to delete vaulted payment method")
+                                        exp.fulfill()
                                     } else {
```

Comment:
1: This assertion I believe needed to be flipped- the test now passes.
2: With the expectation outside of these callbacks as it was, we never got this far to test the assertion.

@BorisNikolic please confirm my assumptions 🙏🏻

Reply:
`updatedVaultedPaymentMethods` should return the mocked result per line 331 whcih is an empty array, so yes this is right afaict.